1-estrategia/README.md

```diff
@@ -1,23 +1,237 @@
+
+- En el apartado de “atención al cliente”, es imprescindible añadir un **e-mail** para que los clientes se puedan poner en **contacto**, es la forma más demandada. 
+
+
+## Personas
+
+Mediante el uso de personas vamos a representar a los posibles personajes que van a usar el sitio web que vamos a diseñar. De esta forma no solo representamos a una persona sino al conjunto de las mismas, recogemos sus conductas, habilidades, actitudes de nuestros posibles usuarios potenciales. Este proceso nos va a ayudar mucho a la hora de elaborar nuestra interfaz ya que diseñaremos la misma en función de las necesidades de nuestros usuarios. 
+Para ello hemos elaborado tres plantillas y por ende tres personas que representan al conjunto de nuestro posible público objetivo. Ellos son: Juan, Paula y Dolores. 
+
+* **Juan** es estudiante de criminología y este año termina la carrera. Dice no tener problemas a la hora de manejar diferentes sitios webs pero tampoco de considera un hacha.
+* **Paula** es directora informática de BOSCH Bilbao. Le apasiona estar en todo momento informada y le encanta leer sobre temas de desigualdad de género. No tiene ningún tipo de problema a la hora de manejar sitios web, ya que se dedica a ello. 
+* **Dolores** es ama de casa y no controla mucho el mundo web. Se siente segura utilizando la página web de las recetas de MJ ya que le resulta familiar y sencilla. Por lo demás prefiere no tocar mucho no vaya a des configurar el ordenador que tiene en casa. Para estar informada compra todos los días prensa escrita pero no le importaría pasarse plataformas web siempre y cuando sean claras y sencillas.
+
+* <img src="img/JUAN.jpg">
+* <img src="img/PAULA.jpg">
+* <img src="img/DOLORES.jpg">
+
+Hemos decidido que vamos a diseñar nuestra interfaz para Dolores ya que es quién mas problemas va a tener a la hora de navegar en nuestro sitio web. Paula y Juan están muy familiarizados con este entorno y creemos que no tendrán ningún tipo de problema a la hora de navegar en nuestra web. 
 
-### Personas
+
+### Lean UX Canvas
 
 ## Lean UX Canvas
 
-(plantilla/modelo en el documento `lean-ux-canvas.md`)
 
-## Referencias
\ No newline at end of file
+
+<table markdown="1"><tbody><tr><td markdown="1">
+
+## LEAN UX-CANVAS
+
+**Problema**
+
+Hemos visto que la audiencia ya no se suscribe a las páginas web como antes. Buscan únicamente un contenido concreto y se van. ¿Cómo podemos optimizar para lograr una identidad por su parte hacia nuestro sitio?
+
+</td><td rowspan=2 markdown="1">
+
+**Ideas de soluciones**
+
+Sumar todo tipo de tecnologías a nuestra mano para hacer la experiencia de usuario más dinámica.
+
+Utilizar el fenómeno transmedia como motor de vehículo de nuetsra web (sumando podcast, video reportajes, artículos...).
+
+Dar voz a nuestro usuarios a través de comentarios, encuestas...
+
+Crear una comunidad participativa a través de informaciones, pero también de vivencias personales.
+
+</td><td markdown="1">
+
+**Resultados**
```

Comment:
Este apartado debería derivarse de los indicadores de éxito. No están aquí todos los que habéis mencionado allí, y viceversa.